run-tests-with-build-drools-parent/drools-project-sources-test/pom.xml

```diff
@@ -46,35 +46,42 @@
           </execution>
         </executions>
       </plugin>
-
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-invoker-plugin</artifactId>
-        <configuration>
-          <parallelThreads>1</parallelThreads>
-          <!--
-          script deciding if included project is:
-            executed (either script missing or returning true)
-              or
-            skipped (when returns false or throws error) -->
-          <selectorScript>${maven.modules.resolution.selector.script.name}</selectorScript>
-          <pomIncludes>
-            <!-- includes everything, filtering is done in resolve-includes.groovy script
-            which adds invoker-run.groovy script containing 'return false' near each pom.xml
-            that should be excluded.-->
-            <pomInclude>**/pom.xml</pomInclude>
-          </pomIncludes>
-          <pomExcludes>
-            <!-- Excludes work still as expected. -->
-            <pomExclude>**/kie-archetypes/kie-drools-archetype/pom.xml</pomExclude>
-            <pomExclude>**/kie-archetypes/kie-drools-archetype/src/main/resources/archetype-resources/pom.xml</pomExclude>
-            <pomExclude>**/kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/archetype-resources/pom.xml</pomExclude>
-            <pomExclude>**/kie-archetypes/kie-drools-exec-model-archetype/src/main/resources/archetype-resources/pom.xml</pomExclude>
-            <pomExclude>**/kie-archetypes/kie-drools-exec-model-ruleunit-archetype/src/main/resources/archetype-resources/pom.xml</pomExclude>
-            <pomExclude>**/drools-test-coverage/standalone/kie-ci-with-domain/tests/pom.xml</pomExclude>
-            <pomExclude>**/drools-test-coverage/standalone/kie-ci-without-domain/tests/pom.xml</pomExclude>
-          </pomExcludes>
-        </configuration>
+        <executions>
+            <execution>
+            <id>run-tests</id>
+            <configuration>
+              <goals>
+                <goal>clean</goal>
+                <goal>org.kie:unpack-build-maven-plugin:1.0-SNAPSHOT:unpack-build</goal>
+                <goal>verify</goal>
+              </goals>
+              <parallelThreads>1</parallelThreads>
+              <properties>
```

Comment:
The parallelThread and properties elements are not inherited from the dependencyManagement?